webapp/pages/post/_id/_slug/index.vue

```diff
@@ -122,7 +132,7 @@ export default {
     UserTeaser,
     HcShoutButton,
     ContentMenu,
-    HcCommentForm,
+    CommentForm,
```

Comment:
:+1: I like this!

Reply:
yeah, I opened a ticket to rename all these in bulk, but I've been sneaking some of these in cause I can't help myself